File: uri/principiante/multiples_of_13/1132.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main() {
    int x, y;
    cin >> x >> y;
    
    int minVal = min(x, y);
    int maxVal = max(x, y);
    
    int sum = 0;
    
    for (int i = minVal; i <= maxVal; i++) {
        if (i % 13 != 0) {
            sum += i;
        }
    }
    
    cout << sum << endl;
    
    return 0;
}

File: uri/principiante/multiples_of_13/1132.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let x: i32 = input.trim().parse().unwrap();
    
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let y: i32 = input.trim().parse().unwrap();
    
    let min_val = x.min(y);
    let max_val = x.max(y);
    
    let mut sum = 0;
    
    for i in min_val..=max_val {
        if i % 13 != 0 {
            sum += i;
        }
    }
    
    println!("{}", sum);
}
